frontend/src/components/common/form/Typeahead.tsx

```diff
@@ -92,7 +106,9 @@ export function TypeaheadField<T extends TypeaheadModel>({
                 setFieldValue(name, getIn(newValues[0], 'value') ?? newValues[0]);
               }
         }
-        onBlur={() => setFieldTouched(name, true)}
+        onBlur={
+          selectClosest ? (e: any) => handleBlur(e.target.value) : () => setFieldTouched(name, true)
```

Comment:
FYI formik has a handleBlur function that is a bit shorter then using setFieldTouched

Reply:
May also be a reason not to use handleBlur as the name of your custom function to avoid confusion.

Reply:
Ooh good point, will adjust